doc/api/url.md

```diff
@@ -313,20 +313,39 @@ myURL.port = 1234.5678;
 console.log(myURL.port);
 // Prints 1234
 
-// Out-of-range numbers are ignored
-myURL.port = 1e10;
+// Out-of-range numbers, which are not represented in scientific noation,
+// will be ignored.
+myURL.port = 1e10; // 10000000000, will be range-checked as described below
 console.log(myURL.port);
 // Prints 1234
+
 ```
 
 The port value may be set as either a number or as a String containing a number
 in the range `0` to `65535` (inclusive). Setting the value to the default port
 of the `URL` objects given `protocol` will result in the `port` value becoming
 the empty string (`''`).
 
-If an invalid string is assigned to the `port` property, but it begins with a
-number, the leading number is assigned to `port`. Otherwise, or if the number
-lies outside the range denoted above, it is ignored.
+Upon assigning a value to the port, the value will first be converted to a string using `.toString()`.
+
+If that string is invalid, but it begins with a
+number, the leading number is assigned to `port`.  
+Otherwise, or if the number
+lies outside the range denoted above, it is ignored.  
+
+Note that numbers which contain a decimal point,  
+such as floating-point numbers or numbers in scientific notation, are not an exception to this rule.  
+Leading numbers up to the decimal point will be set as the url's port, assuming they are valid. 
+
+For example:
+
+```js
+myURL.port = 4.567e21;
+console.log(myURL.port);
+// Prints 4 (because it is the leading number in the string "4.567e21")
```

Comment:
Nit: `"4.567e21"` -> `'4.567e21'`?
We usually use single quotes in code-related things.